aten/src/ATen/core/dispatch/DispatchKeyExtractor.h

```diff
@@ -101,6 +101,10 @@ struct CAFFE2_API DispatchKeyExtractor final {
     return DispatchKeyExtractor(schema.arguments().size());
   }
 
+  static DispatchKeyExtractor makeUninitialized() {
+    return DispatchKeyExtractor(0);
```

Comment:
This means there is no way to distinguish from an op actually taking zero arguments and an uninitialized DispatchKeyExtractor. It's probably fine though.

Reply:
Yes, I don't think it causes any problems.